### ISSUE TITLE
[FEATURE] Add ARIA attributes to faq section to increase accessibility

### ISSUE THREAD

**Issue by Avinash905** [devs-in-tech/DevsInTech#391]
### What would you like to share or ask?

**Description**
ARIA attributes are crucial for enhancing web application accessibility, benefiting users with disabilities. Currently, the faq section lacks these attributes, hindering proper interpretation by assistive technologies.

**Some steps to implement this enhancement-**

- Identify the accordian elements in your codebase.
- Determine and add the relevant ARIA attributes to add for each element.
- Include concise and descriptive ARIA labels using the aria-label attribute.
- Use the aria-describedby attribute to provide additional instructions for these elements.
- Implement ARIA states and properties (e.g., aria-required for required fields).
- Test the accessibility improvements using assistive technologies.

**Comment by Avinash905** [comment]
Please assign me this issue so that I can work on this issue under **GSSOC'23**.
